### PR TITLE
Changing the commit message to 1000 characters

### PR DESCRIPTION
**Pull request by shpraka**
This covers #288. 

Recently we found out that 7000 doesn't work in some scenarios intermittently, keeping it a bit lower for the deployment to go through.

### CODE REVIEW

**Review by kumaraksh1 — APPROVED** [review]
LGTM!!

**Comment by AaronL1011** [comment]
Any plans to merge this one? This issue is intermittently blocking our pipelines, would love to see progress 🙏

**Comment by shpraka** [comment]
@AaronL1011 Could you share the debug logs of the failed runs? I had tried reproducing same last week and it wasn't failing for me with existing limit?

**Comment by Methuselah96** [comment]
Added an example of it failing in https://github.com/Azure/webapps-deploy/issues/288.

File: lib/actionparameters.js
Comment (AaronL1011) [inline lib/actionparameters.js:40]: Given @IGx89's point in issue https://github.com/Azure/webapps-deploy/issues/288 on URL encoding and how it will largely increase the length of the zipdeploy URL post-encode, should this trim be closer to 1000? Ditto for the other file change 👍

**Review by kumaraksh1 — APPROVED** [review]
LGTM !!

**Comment by IGx89** [comment]
This looks good to me too, and appears like it should resolve #288. Still waiting on something before merging?

**Comment by AaronL1011** [comment]
Yeah keen to get this one on the move, is there a hold up in the deploy pipeline process? Not sure who is the right person to push this through.

**Review by kumaraksh1 — APPROVED** [review]
LGTM!!

**Review by kumaraksh1 — APPROVED** [review]
LGTM!!

**Review by shubham805 — APPROVED** [review]
LGTM

**Comment by shpraka** [comment]
This PR is covered as part of #301.

**Pull request closed without merging**
